components/performance-chart/styles.css

```diff
@@ -22,7 +23,6 @@
   position: absolute;
   top: 50%;
   left: 50%;
-  margin: 12.5% 0 0;
```

Comment:
It's calculated value from dropped 25% in the parent container. Why do you remove this?

Reply:
It looked good on small screen but when screen gets bigger it started to move down to the chart. So I rather removed it.